codegen/settingsgen.py

```diff
@@ -262,7 +272,7 @@ def _populate_classes(parent_dir):
                 f.write(f"{istr2}{mn}\n\n")
 
                 for child in child_names:
-                    f.write(f"{istr1}{child}: {child} = {child}\n")
+                    f.write(f"{istr1}{child}: {child}_cls = {child}_cls\n")
```

Comment:
Generating
```
energy: energy_cls = energy_cls
```
instead of
```
energy: energy = energy
```
The type-hint was getting resolved to the `energy` attribute instead of the `energy` class in the previous version.